trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.slice(0, 10)
```

Comment:
- [ ] This logic of trimming can be modified by a condition function, so basically, we want to tell the class to only trim the name if it exceeds 10 characters. Please implement the correct logic for this. 👍